### DISCUSSION TITLE
Better Carousel For Quasar

### DISCUSSION THREAD

**Discussion by mrmmg** [quasarframework/quasar#9655] [CLI - PWA mode] [answered]
Hi

I'm Developing PWA with Quasar (V1.15.20) and I want to create a carousel slider. I used this component [Carousel Component]( https://quasar.dev/vue-components/carousel) but it's not good for my project. I want a better carousel library. I tested some libraries from npmjs.org but those libraries do not work with Quasar, for example, this: [SplideJS](https://splidejs.com)

Are you know a good library that works with Quasar completely?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@mrmmg 

> Are you know a good library that works with Quasar completely?

I would look for carousels made in Vue. Btw othen for populair non vue libraries a vue wrapper version is available,
for example Vue SpileJS:
https://github.com/Splidejs/vue-splide

**Reply by liconver** [reply]
So I tried adding splide into the boot and the app wouldn't build.

boot/splide.js 
`
   import { boot } from "quasar/wrappers";
   import { VueSplide } from "@splidejs/vue-splide";

  export default boot(async ({ app }) => {
  app.use(VueSplide);
});
`

config
`
    boot: ["axios", "splide"],

 `

How would you get splide to work with quasar?

**Reply by yusufkandemir** [reply]
@liconver https://splidejs.com/integration/vue-splide/#registration
you should import it as `import VueSplide from "@splidejs/vue-splide";` using the default export. Don't include `{  }`.

You would also need to import some CSS files in that boot file too: https://splidejs.com/integration/vue-splide/#css

**Reply by liconver** [reply]
Thanks I got it working I thought because I re-ran yarn quasar dev but I guess it was because of the { }, because without the braces it uses the default export from the module.
